.prow.yaml

```diff
@@ -145,6 +145,8 @@ presubmits:
           value: "aws"
         - name: CONTAINER_RUNTIME
           value: containerd
+        - name: TF_VAR_initial_machinedeployment_spotinstances
+          value: "true"
```

Comment:
This looks okay to me, but we also need to change periodics.

An alternative approach would be that we apply this flag when provisioning the cluster, such as [here](https://github.com/kubermatic/kubeone/blob/a6096111313da857cfafd64c1a8b035df7b2c821/test/e2e/conformance_test.go#L199-L201) and [here](https://github.com/kubermatic/kubeone/blob/a6096111313da857cfafd64c1a8b035df7b2c821/test/e2e/upgrade_test.go#L179-L181). In that case, we don't need to change all jobs.

Reply:
In indicated places we are forced to have hacks like this because we have to dynamically change variable. In general such configs should be explicitly shown.